tests/foreman/ui/test_products.py

```diff
@@ -103,19 +88,20 @@ def test_negative_create_with_blank_name(self, name):
 
         """
         with Session(self.browser) as session:
-            make_product(
-                session,
-                org=self.organization.name,
-                loc=self.loc.name,
-                name=name,
-                description=gen_string('alphanumeric'),
-            )
-            self.assertIsNotNone(self.products.wait_until_element(
-                common_locators['common_invalid']))
+            for name in invalid_values_list(interface='ui'):
+                with self.subTest(name):
+                    make_product(
+                        session,
+                        org=self.organization.name,
+                        loc=self.loc.name,
+                        name=name,
+                        description=gen_string('alphanumeric'),
+                    )
+                    self.assertIsNotNone(self.products.wait_until_element(
+                        common_locators['common_invalid']))
 
     @run_only_on('sat')
-    @data(*generate_strings_list())
-    def test_negative_create_with_same_name(self, prd_name):
```

Comment:
I intentionally changed this from datadriven to normal test as repeating this test for all generate_strings_list() is redundant.